code/game/objects/buckling.dm

```diff
@@ -117,11 +117,71 @@
 /atom/movable/proc/post_unbuckle_mob(mob/living/M)
 	return
 
+/atom/movable/proc/user_buckle_minorfail(mob/living/M, mob/user)
```

Comment:
I can't say I'm a fan of having two separate procs for a minor and major fail. Perhaps you could mimic the severity system EMPs and explosions have, using numbers?

```dm
/atom/movable/proc/user_buckle_fail(mob/living/M, mob/user, severity)
	switch(severity)
		if(1)
			// Minor fail
		if(2)
			// Major fail
		if(3)
			// target.dust()
```